src/OpenTelemetry/Logs/ILogger/OpenTelemetryLoggerOptions.cs

```diff
@@ -58,13 +58,21 @@ public class OpenTelemetryLoggerOptions
     /// <remarks>
     /// Notes:
     /// <list type="bullet">
-    /// <item>Parsing is only executed when the state logged does NOT
-    /// implement <see cref="IReadOnlyList{T}"/> or <see
+    /// <item>As of OpenTelemetry 1.5.0 state parsing is handled automatically
```

Comment:
We might want to change it to `1.5.1`? This might lead to users look specifically for 1.5.0 .

```suggestion
    /// <item>As of OpenTelemetry 1.5.1 state parsing is handled automatically
```

Reply:
I updated it to be "v1.5" so it isn't tied to any specific patch version. How do you feel about that?

Reply:
Looks good!